x-pack/plugins/security_solution/server/endpoint/routes/policy/handlers.test.ts

```diff
@@ -319,6 +319,25 @@ describe('test policy response handler', () => {
         sortField: undefined,
       });
     });
+
+    it('should pass other query params as expected', async () => {
+      const mockRequest = httpServerMock.createKibanaRequest({
+        query: { page: 1, pageSize: 11 },
+      });
+
+      await policyHandler(
+        createRouteHandlerContext(mockScopedClient, mockSavedObjectClient),
+        mockRequest,
+        mockResponse
+      );
+      expect(mockPackagePolicyService.list.mock.calls[0][1]).toEqual({
```

Comment:
I am curious to know why you prefer this syntax over  `expect(mock).toHaveBeenCalledWith()'